packages/adapter-firebase/src/index.ts

```diff
@@ -115,7 +115,7 @@ export interface FirebaseAdapterConfig extends AppOptions {
  * })
```

Comment:
@balazsorban44 the exported util functions which are not the main adapter function, end up a bit in the middle of the docs: 

<img width="800" alt="image" src="https://user-images.githubusercontent.com/5938217/224483983-4a8e48b4-f68d-4042-8fca-7b47d73e0228.png">

Would be good to see if we can change the **Typedoc** plugin and have more control over where these are placed... For me, these should go under **Functions** 💭

Reply:
This is an indentation error. `Basic usage` and everything below should be +1 indented, so `FirestoreAdapter` and `initFirestore` end up on the same level.